build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.2.0'
+        classpath 'com.android.tools.build:gradle:2.2.2'
```

Comment:
I was being conservative and staying on the stable channel. The beta channel builds have been expiring, so instead of chasing after those versions, it might be best to stick until AS 2.3 is final. Happy to bump if you disagree.